eliHenderson/assignments/twopointers/w1d3/lc15.js

```diff
@@ -0,0 +1,45 @@
+const search_triplets = function (arr) {
+  arr.sort();
+  let results = [];
+  let n = 0;
+  var l = arr.length;
+  for (var i = 0; i < l; i++) {
+    // move i to the next unique elem
+    if (i > 0 && arr[i] === arr[i - 1]) {
+      // skip same element to avoid duplicate triplets
+      continue;
+    }
+    var j = i + 1,
```

Comment:
`var` is no longer recommended to use. Alternatives would be `const` and `let`.

Reply:
Yeah, when I was practicing this problem, the solution I  found had var in it. I guess I subconsciously memorized var. I generally never use it.